.github/workflows/modrinth-publish.yml

```
@@ -0,0 +1,26 @@
+name: Automatically publish to Modrinth and CurseForge
+on:
+  release:
+    types: [published]
+jobs:
+  publish-to-modrinth:
+    runs-on: ubuntu-latest
+    steps:
+       - name: Get the version
+         id: get_version
+         run: echo ::set-output name=VERSION::${GITHUB_REF/refs\/tags\//}
+       - name: Download the pack files
+         run: |
+           curl -so "MR_Fabulously Optimized_${{ steps.get_version.outputs.version }}.mrpack" https://github.com/Fabulously-Optimized/fabulously-optimized/releases/download/${{ steps.get_version.outputs.version }}/MR_Fabulously.Optimized_${{ steps.get_version.outputs.version }}.mrpack
```

```suggestion
           curl -so "MR_Fabulously.Optimized_${{ steps.get_version.outputs.version }}.mrpack" https://github.com/Fabulously-Optimized/fabulously-optimized/releases/download/${{ steps.get_version.outputs.version }}/MR_Fabulously.Optimized_${{ steps.get_version.outputs.version }}.mrpack
```
GitHub seems to replace spaces with dots

---

Shouldn't be an issue since this is output file, I've changed this so it actually works anyway